packages/dataviews/src/style.scss

```diff
@@ -809,6 +825,38 @@
 	}
 }
 
+/* stylelint-disable-next-line scss/at-rule-no-unknown -- '@container' not globally permitted */
+@container (max-width: 430px) {
```

Comment:
@jameskoster @youknowriad @jasmussen 

![badge](https://img.shields.io/badge/review-ask-blue.svg) Would it make sense to update this to `$break-mobile: 480px` instead of the hard-coded `430px`? Currently, `430px` is hard-coded outside of DataViews, which means third-party apps using DataViews might need to replicate this hard-coding if they want to align their padding with DataViews ([an example](https://github.com/Automattic/wp-calypso/pull/96764/files#diff-2d032de0d29884aba6731f1d763ee6159a7ed5d1d6df7e0b70b2560f2ab3cecaR74)). 
If not, do you have plans to define `430px` as a reusable variable? Thank you!

Reply:
The question seems to be what makes sense for DataViews component when used in isolation (storybook can be a good way to think about this).

Reply:
Right, I think the instinct to find a _rule_ that makes this value feel less arbitrary is valid enough in terms of the component system. But honestly I'd love to move all of the UI towards something better than the 480/600/782px values that currently make up the breakpoints. Those seem perhaps even more arbitrary.